opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -111,4 +122,19 @@
      * @return this Span instance, for chaining
      */
     Span setOperationName(String operationName);
+
+    /**
+     * @deprecated use {@link #log(Map)} like this
+     * {@code span.log(Map.of("event", "timeout"))}
```

Comment:
basically, we have a responsibility to think about general implementors. currently our option is to assume a single-entry map should be rendered as a plain string. The problem with axing the plain api is that now we have to make guesses for the most form of trace annotations: a simple string.

If we make a constant, we don't pollute existing systems with {"event":"foo"}, or require users of the generic api to always read the notes of each tracer impl when trying to figure out how to log a simple string